modules/aws/ec2_test.go

```diff
@@ -31,3 +35,162 @@ func TestGetEc2InstanceIdsByFilters(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, 0, len(ids))
 }
+
+func TestGetRecommendedInstanceType(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		region              string
+		instanceTypeOptions []string
+	}{
+		{"eu-west-1", []string{"t2.micro", "t3.micro"}},
+		{"ap-northeast-2", []string{"t2.micro", "t3.micro"}},
+		{"us-east-1", []string{"t2.large", "t3.large"}},
+	}
+
+	for _, testCase := range testCases {
+		// The following is necessary to make sure testCase's values don't
+		// get updated due to concurrency within the scope of t.Run(..) below
+		testCase := testCase
```

Comment:
Consider linking to the Go docs on this. I think it's here: https://golang.org/doc/faq#closures_and_goroutines - I found this to be helpful helpful previously when reading the Terragrunt source.

Reply:
Done: https://github.com/gruntwork-io/terratest/pull/577/commits/81268c2e09dc16ac414445c2ca886edc02fd2a2e